fuseimg/data/ops/image_loader.py

```diff
@@ -35,3 +35,33 @@ def __call__(self, sample_dict: NDict, op_id: Optional[str], key_in:str, key_out
     def reverse(self, sample_dict: dict, key_to_reverse: str, key_to_follow: str, op_id: Optional[str]) -> dict:
         return sample_dict
 
+class OpLoadDicom(OpBase):
+    '''
+    Loads a medical image, currently only nii is supported
+    '''
+    def __init__(self, dir_path: str , **kwargs):
+        super().__init__(**kwargs)
+        self._dir_path = dir_path
+
+    def __call__(self, sample_dict: NDict, key_in:str, key_out: str, format:str="infer"):
+        '''
+        :param key_in: the key name in sample_dict that holds the filename
+        :param key_out: 
```

Comment:
Do you extract just the image?
Do support an option to read more fields from the dicom? (it will be useful)

Reply:
I added basic field reading